src/main/java/money/Goal.java

```diff
@@ -0,0 +1,61 @@
+package money;
+
+import java.time.LocalDate;
+import java.util.Date;
+
+public class Goal extends Expenditure {
+
+    public enum Priority {
+        HIGH,
+        MEDIUM,
+        LOW
+    }
+
+    private Priority priority;
+
+    public Goal(float price, String description, String category, LocalDate goalBy, String priorityLevel) {
+        super(price, description, category, goalBy);
+        switch (priorityLevel) {
+        case "LOW" : {
+            this.priority = Priority.LOW;
+            break;
+        }
+        case "MEDIUM": {
+            this.priority = Priority.MEDIUM;
+            break;
+        }
+        case "HIGH": {
+            this.priority = Priority.HIGH;
+            break;
+        }
+        default: {
+            break;
+        }
```

Comment:
Good attempt at using enums. Take a look at the documentation for enum and you'll find a more effective way of implementing this :)